OJP/OJP_Trips.xsd

```diff
@@ -106,6 +107,18 @@
 			</xs:element>
 		</xs:sequence>
 	</xs:group>
+	<xs:complexType name="AccessModesListOfEnumerations">
+		<xs:annotation>
+			<xs:documentation>List of Access Modes.</xs:documentation>
+		</xs:annotation>
+		<xs:sequence>
+			<xs:element name="AccessMode" type="ifopt:AccessModesEnumeration" minOccurs="0">
```

Comment:
Is a *maxOccurs="unbounded"* missing here? And couln't the term *List* be misleading, since it wouldn't be a real list where multiple items are possible between the element tags (*<<AccessModeList>>ski car bicycle<</AccessModeList>>*)?

Reply:
yes, it should.

Reply:
done